### ISSUE TITLE
[Feature request] : Specific shape transform (like inkscape)

### ISSUE THREAD

**Issue by bobpatraque** [MaurycyLiebner/enve#121]
Ih there,

I would like to use enve to do some 2d motion graphic, and I would like to know if adding the ability to use specific shape transformation was possible.

As I'm not 100% sure I'm clear in my request, I link a picture of what I would love to do in enve with shapes like circle or stars (done with inkscape).

![Shape Tooling](https://user-images.githubusercontent.com/46961681/80959142-77dad700-8e06-11ea-8687-71f9324d0396.jpeg)

I also link an aftereffect tutorial to do a looping loading circle, that can be done with that kind of "tool".

https://www.youtube.com/watch?v=EmTlTjSEVcA

I hope I'm clear enough to be understandable, and that my feature request is'nt irrelevant.

Thank you for your hard work on enve !

**Comment by arifination** [comment]
yes, this is basic feature even for me (coreldraw user) some time it's very usefull. :D
